packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature

```diff
@@ -8,13 +8,15 @@ Feature: Sign In with Force New Password flow
     Given I'm running the example "ui/components/authenticator/sign-in-with-phone"
 
   Scenario: Sign in using a valid phone number and password and user is in a FORCE_CHANGE_PASSWORD state
-    When I type my "phone number" with status "FORCE_CHANGE_PASSWORD"
+    When I select my country code with status "FORCE_CHANGE_PASSWORD"
```

Comment:
Why is status necessary for selecting a country code?

Reply:
Country code is my method of determining which status a user with a given phone number has in Cognito. Since I reduced our environment variables to a single "alias", we needed some way of programmatically signing in with a phone number that corresponds to a Cognito status in our e2e like we do for username and email.